src/elements/Button/Button.tsx

```diff
@@ -79,7 +92,11 @@ export const Button: FC<ButtonProps & ButtonRef> = forwardRef(
         [css[variant]]: true
       })}
     >
-      {children}
+      <Spacer dense>
+        {start && start}
```

Comment:
This needs to be explicit undefined check - what if start = 0

Reply:
Do we need a CSS class for these start end? Whats the advantage of that over just putting them in children beside each other?

Reply:
mm i don't think so - I feel like if a class is needed, it can just be part of what's passed in as part of the `start` and `end` props, no?